CK2Plus_expanded/common/cultures/000_CK2Plus_iberian.txt

```diff
@@ -141,7 +141,7 @@ iberian = {
 
 	leonese = { 
 		graphical_cultures = { southerngfx }
-		color = { 0.45 0.75 0.35 }
+		color = { 0.78 0.47 0.85 }

```

Comment:
I just thought this would be a decent change. It's basically Leonese purple, and it makes it way easier to distinguish from Andalusian culture on the map.